### PR TITLE
basic Error modal

### PR DESCRIPTION
**Pull request by fanlan1210**
aka Disconnect/Error Screen/Handler in the TODO.

### CODE REVIEW

File: sites/game-client/client.css [hunk 1]
```diff
@@ -132,3 +132,17 @@ body {
...
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+#error-modal > div {
+  width: 50%;
+}
```
Comment (john0312) [inline sites/game-client/client.css:148]: Optional: Add new line.

File: sites/game-client/ui/error-modal.mjs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 HITCON Online Contributors
+// SPDX-License-Identifier: BSD-2-Clause
+
+import Modal from './modal.mjs';
+
+const ERROR_MODAL_DIV = 'error-modal';
+const ERROR_MESSAGE_DIV = 'error-message';
+const ERROR_HANDLER_MESSAGE_DIV = 'error-handler-message';
+
+class ErrorModal extends Modal {
```
Comment (john0312) [inline sites/game-client/ui/error-modal.mjs:10]: Optional: Add brief class and function comment.

**Review by john0312 — APPROVED** [review]
LGTM with optional suggestions.

File: sites/game-client/client.css (hunk 1 above)
Comment (fanlan1210) [reply sites/game-client/client.css]: Added.

**Pull request merged**